nca/NetworkConfig/NetworkLayer.py

```diff
@@ -252,6 +253,17 @@ def collect_policies_conns_optimized(self, is_ingress, captured_func=lambda poli
         """
         res_conns = OptimizedPolicyConnections()
         for policy in self.policies_list:
+            # Track the peers that were affected by this policy
+            if ExplTracker().is_active():
+                for peer in policy.selected_peers:
+                    src_peers, _ = ExplTracker().extract_peers(policy.optimized_allow_ingress_props)
+                    _, dst_peers = ExplTracker().extract_peers(policy.optimized_allow_egress_props)
```

Comment:
this can be added in the stage of adding the policy from `PoliciesFinder`, since optimized connectivity properties are already computed during parsing.

Reply:
Done